partialWaveFit/fitResult.cc

```diff
@@ -387,34 +401,49 @@ fitResult::fill(const fitResult& result,
                 const complexMatrix*            acceptedNormIntegral,    // normalization integral matrix with acceptance
                 const vector<double>*           phaseSpaceIntegral)      // normalization integral over full phase space without acceptance
 {
-	std::vector<std::complex<double>> prodAmpsEmpty;
+	if (fitParCovMatrix == nullptr)
+		fitParCovMatrix = (not result.covMatrixValid()) ? nullptr : &result.fitParCovMatrix();
+	if (fitParCovMatrixIndices == nullptr)
+		fitParCovMatrixIndices = &result.fitParCovIndices();
```

Comment:
Why is `fitParCovMatrixIndices` still a pointer in this function?

Reply:
It should not! Thank you.